web/lumpTypeView.ts

```diff
@@ -471,3 +471,62 @@ function drawMapGeometry(
         }
     }
 }
+
+export function LumpTypeViewPlaypal(
+    options:{scaleX:number, scaleY:number} = {scaleX: 4, scaleY: 4}
+) {
+    return new LumpTypeView({
+        name: "Playpal",
+        icon: "assets/icons/lump-playpal.png",
+        view: (lump: WADLump, root: HTMLElement) => {
+            const container2 = util.createElement({class: "lump-view-playpal", appendTo: root});
+            const container = util.createElement({class: "lump-view-playpal-inner", appendTo: container2});
+            const numPals = Math.floor(lump.dataLength / 768); // Sometimes, the PLAYPAL has a comment at the end.
+            let curPal = 0;
+            const nextArrow = util.createElement({content: "»", class: "lump-view-playpal-navbtn", onleftclick: () => {
+                    curPal += 1;
+                    if (curPal >= numPals) { curPal = numPals - 1; }
+                    drawPalette(curPal);
+                }
+            });
+            const prevArrow = util.createElement({content: "«", class: "lump-view-playpal-navbtn", onleftclick: () => {
+                    curPal -= 1;
+                    if (curPal < 0) { curPal = 0; }
+                    drawPalette(curPal);
+                }
+            });
```

Comment:
Please change the above lines (L492 through L497) to this:

``` ts
            const prevArrow = util.createElement({
                content: "«",
                class: "lump-view-playpal-navbtn",
                onleftclick: () => {
                    curPal -= 1;
                    if (curPal < 0) { // I like this better as wrapping to end instead of clamping at start
                        curPal = numPals - 1;
                    }
                    drawPalette(curPal);
                },
            });
```